voter-dapp/src/ActiveRequests.js

```diff
@@ -251,22 +252,25 @@ function ActiveRequests({ votingAccount, votingGateway }) {
     }
   });
 
-  const canExecuteBatch = () => {
+  const canExecuteBatchCommit = () => {
     let totalSelected = 0;
     for (let checked in checkboxesChecked) {
       totalSelected += checkboxesChecked[checked];
     }
-    // This number defines the maximum number of transactions that can be fit within one block.
-    // It was calculated by testing the batchCommit and batch revealFunctions to identify the
-    // maximum that can be placed within one tx. The actual number that can be fit is slightly
-    // more but is set to a lower amount for safety here.
-    return totalSelected <= 25;
+    return totalSelected <= BATCH_MAX_COMMITS;
+  };
+  const canExecuteBatchReveal = () => {
```

Comment:
nit: just to make this a little more DRY, wdyt about just making one function and pass the limit in as an argument?

Reply:
done